src/pages/consent/consent-page.ts

```diff
@@ -30,28 +25,32 @@ export class ConsentPage extends LitElement {
     this.descriptionField.value = this.description;
   }
 
+  getChangeLink() {
+    if (Object.keys(WAKE_WORDS).length < 2) return;
+    return html`&nbsp;(<a href="#" @click=${this.cancelConsent}>change</a>)`;
+  }
+
   render() {
     if (this.givingConsent) {
```

Comment:
We can remove this boolean, as this is now on the next page.

Reply:
You marked this one as resolved, but `givingConsent` is still not used? Shouldn't this whole block be removed. It was only there to ensure that the page changed while we were requesting microphone access